pkg/apis/clusterlink.net/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 The ClusterLink Authors.
```

Comment:
Nit: It's 2024, ...
Need to update the scripts (or the codegen boilerplate under kubebuilder's `hack` directory) to handle that correctly.
Should be done as separate issue/PR #285